builder/public/30s.css

```diff
@@ -119,6 +119,32 @@ p > code {
     margin-left: 15px;
 }
 
+.level-navigation ul {
+    display: flex;
+    justify-content: space-between;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+.level-navigation ul li {
+    flex-grow: 1;
+    text-align: center;
+}
+
+.level-navigation ul li a {
+    display: block;
+    margin: 1rem;
```

Comment:
Can we just use pixels here to be consistent with rest of the codebase?

Reply:
My habit, sorry. Will fix it